db/re/skill_db.yml

```diff
@@ -42803,15 +42801,15 @@ Body:
         Time: 30000
     Duration2:
       - Level: 1
-        Time: 12000
+        Time: 10000
       - Level: 2
-        Time: 14000
+        Time: 12000
       - Level: 3
-        Time: 16000
+        Time: 14000
       - Level: 4
-        Time: 18000
+        Time: 16000
       - Level: 5
-        Time: 20000
+        Time: 18000
```

Comment:
It's interesting that this has a different duration per level while the original Sandman trap does not.

Reply:
I don't have the 100 loyalty points to test this (You need it in order to get the level 8 Bowman Mercenary which has the Sandman skill). We've had this status duration for years so I doubt it's changed if that's what was leaked for it before.

Reply:
Probably! Because I couldn't test Mercenaries either, I didn't touch their skills. But  maybe we should just make them the same as the original skills.